Home.md

```diff
@@ -1,40 +1,51 @@
-## Introduction
-[d:swarm](http://dswarm.org) is a **data management platform** that is based on the concept of [community sharing](http://en.wikipedia.org/wiki/Sharing). It is used for the lossless transformation of data from heterogeneous sources into a flexible ([elastic](http://en.wikipedia.org/wiki/Elasticity_%28data_store%29)), data model that can be explicitly made available as [(Linked) Open Data](http://en.wikipedia.org/wiki/Linked_data) (LOD).
+[d:swarm](http://dswarm.org) is a **data management platform** that can be used for the lossless transformation of data from heterogeneous sources into a flexible ([elastic](http://en.wikipedia.org/wiki/Elasticity_%28data_store%29)), data model. This data model can serve as a single source for providing  [Linked Open Data](http://en.wikipedia.org/wiki/Linked_data) (LOD).

```

Comment:
Maybe we should put the d:swarm logo etc. upfront (i.e. more pictures then text at the beginning)